pkg/engine/request_performance_tests/play_game.go

```diff
@@ -0,0 +1,75 @@
+package requestperformancetests
+
+import (
+	"testing"
+
+	"github.com/YetAnotherSpieskowcy/Carcassonne-Engine/pkg/engine"
+	"github.com/YetAnotherSpieskowcy/Carcassonne-Engine/pkg/tilesets"
+)
+
+type TestedRequest func(games *[]engine.SerializedGameWithID, eng *engine.GameEngine, b *testing.B)
+
+func PlayGame(gameCount int, b *testing.B, testedRequest TestedRequest) {
+	b.StopTimer()
```

Comment:
Why is there a StopTimer here?

Reply:
because we want to measure only one of the specified request. We want to ignore any other delays sucha as: game set up, playing turn or getting legal moves. We want to measure how fast are request when called at the begining of the turn. That's why there's call  testedRequest(&games, eng, b), so the inside takes care of measurements. You can that inside tested function are Start and Stop timers

Reply:
Ok, but what specific timer are we stopping here? If I understand correctly, there should be no timer started in this place - each tested request starts and stops timers within itself, and no more timers are started besides that. (ctrl+f finds 5 occurences of StartTimer and 6 of StopTimer). So I still don't understand

Reply:
timer literaly starts at the beginning of a test by default.

Reply:
Ooo, now everything is clear, thanks